entries/i/imgur.com.json

```
@@ -2,7 +2,9 @@
   "Imgur": {
     "domain": "imgur.com",
     "contact": {
-      "twitter": "imgur"
+      "twitter": "imgur",
+      "facebook": "Imgur",
```

```suggestion
      "facebook": "100064745678792",
```

As per the [Facebook test](https://app.circleci.com/pipelines/github/2factorauth/twofactorauth/8433/workflows/0cd3abf9-772f-4937-a62c-ee3ecfbb3289/jobs/8459?invite=true#step-106-6)